lib/rules/multiline-ternary.js

```diff
@@ -18,10 +18,15 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [
+            {
+                enum: ["always", "never"]
+            },
```

Comment:
additionalProperties is wrong for arrays, but isn't there an additionalItems flag we could use instead?